### PR TITLE
Template the Gubernator URL.

### PR DESCRIPTION
**Pull request by spxtr**
Next step is to add this template and the report template into the
config.

Working towards #3158.

### CODE REVIEW

File: prow/plank/controller.go [hunk 1]
```diff
@@ -203,7 +207,13 @@ func (c *Controller) syncJenkinsJob(pj kube.ProwJob) error {
...
-		if url := guberURL(pj, strconv.Itoa(status.Number)); pj.Status.URL != url {
+		pj.Status.BuildID = strconv.Itoa(status.Number)
+		var b bytes.Buffer
+		if err := urlTmpl.Execute(&b, &pj); err != nil {
+			return fmt.Errorf("error executing URL template: %v", err)
+		}
+		url := b.String()
+		if pj.Status.URL != url {
 			pj.Status.URL = url
 			pj.Status.PodName = fmt.Sprintf("%s-%d", pj.Spec.Job, status.Number)
```
Comment (kargakis) [inline prow/plank/controller.go]: Unrelated to this PR but Jenkins jobs are not using pods - is this redundant?

Comment (krzyzacy) [inline prow/plank/controller.go]: It does not affect Jenkins jobs. Though we are moving all Jenkins jobs to Prow.

Comment (spxtr) [inline prow/plank/controller.go]: I think it uses this for live logs. It does feel like it should be unnecessary.

**Pull request merged**